### ISSUE TITLE
`lxc profile list` truncates the profile name that contains space character

### ISSUE THREAD

**Issue by lingxiaoyang** [lxc/lxc#1545]
Sorry that I may not have the latest version installed. Thank you for verifying if this is applicable!

# Required information

 * Distribution: Ubuntu
 * Distribution version: 16.04.2 LTS
 * The output of
   * `lxc-start --version`: 2.0.7
   * `lxc-checkconfig` (not sure if related)
   * `uname -a`: Linux lxyang 4.4.0-75-generic #96-Ubuntu SMP Thu Apr 20 09:56:33 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
   * `cat /proc/self/cgroup` (not sure if related)
   * `cat /proc/1/mounts` (not sure if related)

# Issue description

`lxc profile list` truncates the profile name that contains space character, while `lxc profile create` allows spaces in profile names.

# Steps to reproduce

 1. Step one: create a profile, using a name containing spaces.
 2. Step two: `lxc profile list`

What I have tried:

```console
$ lxc profile create 'my test profile'
Profile my test profile created

$ lxc profile list
default
my

$ lxc profile show my
error: not found

$ lxc profile show 'my test profile'
config: {}
description: ""
devices: {}
name: my test profile 
```

# Information to attach

 - [ ] any relevant kernel output (`dmesg`)
 - [ ] container log (The <log> file from running `lxc-start -n <c> -l <log> -o DEBUG`)
 - [ ] the containers configuration file

**Comment by brauner** [comment]
@lingxiaoyang, this bug needs to be filed against https://github.com/lxc/lxd . I'm closing here and would ask you to re-open over there. :)

**Comment by lingxiaoyang** [comment]
@brauner Thank you!

**Comment by brauner** [comment]
Will be included in the next stable `2.0.10` release which is scheduled to be released this week.
